README.md

```diff
@@ -67,117 +69,171 @@ Install the OAK node from git source:
+First, compile the code from this repo.
 
 ```bash
-# Compile
 git clone https://github.com/OAK-Foundation/OAK-blockchain
-cargo build --release --features neumann-node
+cd OAK-blockchain
+cargo build --release --features turing-node --features dev-queue
+```
+> In order to make local testing easy, use a `dev-queue` flag which will allow for putting a task directly on the task queue as opposed to waiting until the next hour to schedule a task.  This works when the `execution_times` passed to schedule a task equals `[0]`.
 
-# Export genesis state
-./target/release/oak-collator export-genesis-state > genesis-state
+Second, prepare two files, genesis-state and genesis-wasm, for parachain registration.
+```bash
+# Generate a genesis state file
+./target/release/oak-collator export-genesis-state --chain=turing-dev > genesis-state
 
-# Export genesis wasm
-./target/release/oak-collator export-genesis-wasm > genesis-wasm
+# Generate a genesis wasm file
+./target/release/oak-collator export-genesis-wasm --chain=turing-dev > genesis-wasm
+```
 
-# Collator1
+Third, start up the build.
+```bash
 ./target/release/oak-collator \
 --alice \
 --collator \
 --force-authoring \
 --tmp \
+--chain=turing-dev \
 --port 40333 \
 --ws-port 9946 \
 -- \
 --execution wasm \
---chain resources/rococo-local.json \
+--chain ./resources/rococo-local.json \
 --port 30335 \
 --ws-port 9977 
 ```
+After this command you should be able to see the stream output of the node.
+#### Register Turing on Rococo
+1. Navigate to [Local relay sudo extrinsic](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/sudo)
+2. Register your local parachain on the local relay chain by calling `parasSudoWrapper.sudoScheduleParaInitialize` (see the screenshot below). 
+3. Parameters:
+    1. id: **2114**
+    2. genesisHead: switch on "File upload" and drag in the above generated `genesis-state` file.
+    3. validationCode: switch on "File upload" and drag in the `genesis-wasm` file.
+    4. parachain: Yes.
+![image](./media/readme-parachain-registration.png)
+1. Once submitted, you should be able to see the id:2114 from the [Parathread](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/parachains/parathreads) tab, and after a short period on the [Parachains](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/parachains) tab.![image](./media/readme-parachain-post-registration.png)
+2. Once Turing is onboarded as a parachain, you should see block number increases on [Turing explorer](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9946#/explorer).
+
+### 3. Launch Turing Network as a parachain
+This step is optional as you can spin up another Turing Network as the second parachain, but for testing XCM functionality we use another parachain, Mangata, as an example here.
+First, compile the code from Mangata’s repo. Currently due to a bug we need to call cargo build twice, with the second including `--features mangata-rococo`.
```

Comment:
@chrisli30 fyi, I'm unable to reproduce this. The build is working fine for me with the duplicate step. Not sure what I was doing wrong earlier or if this is just fixed on their feature branch.